src/Prelude.hs

```diff
@@ -4,12 +4,12 @@
 
 module Prelude
        ( module Relude
-       , module Colog
-       , module Json
-       , module Time
-       , module Web
+        , module Colog
+        , module Json
+        , module Time
+        , module Web
 
-       , WithLog
+        , WithLog
```

Comment:
:thinking:

Reply:
Ah, sorry for that, forgot to remove some redundant formatting when I was commenting lines...